### PR TITLE
fix(demo): add empty and loading state table demos

### PR DESCRIPTION
**Pull request by mcoker**
* also added primary action container to empty state to support designs

fixes https://github.com/patternfly/patternfly-next/issues/2171

### CODE REVIEW

File: src/patternfly/demos/Table/table-loading-table.hbs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    {{/table-tr}}
+  {{/table-thead}}
+
+  {{#> table-tbody}}
+    {{#> table-tr table-tr--modifier="pf-m-height-auto"}}
+      {{#> table-td table-td--attribute='colspan="8"'}}
+        {{#> bullseye}}
+          {{#> empty-state empty-state--modifier="pf-m-sm"}}
+            {{#> empty-state-icon}}
+              {{#> spinner}}{{/spinner}}
```
Comment (mattnolting) [inline src/patternfly/demos/Table/table-loading-table.hbs]:
```suggestion
              {{#> spinner}}Loading...{{/spinner}}
```

File: src/patternfly/demos/Table/examples/table-loading-demo.hbs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{{#> page page--id="page-layout-table-simple"}}
```
Comment (mattnolting) [inline src/patternfly/demos/Table/examples/table-loading-demo.hbs]:
```suggestion
{{#> page page--id="table-loading-demo"}}
```

File: src/patternfly/demos/Table/examples/table-empty-state-demo.hbs [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{{#> page page--id="page-layout-table-simple"}}
```
Comment (mattnolting) [inline src/patternfly/demos/Table/examples/table-empty-state-demo.hbs]:
```suggestion
{{#> page page--id="table-empty-state-demo"}}
```

File: src/patternfly/demos/Table/table-loading-table.hbs [hunk 4]
```diff
@@ -0,0 +1,43 @@
+{{#> table table--id="empty-loading-demo" table--grid="true" table--modifier="pf-m-grid-xl" table--attribute='aria-label="This is a table showing a loading state"'}}
```
Comment (mattnolting) [inline src/patternfly/demos/Table/table-loading-table.hbs]:
```suggestion
{{#> table table--id="table-loading-table" table--grid="true" table--modifier="pf-m-grid-xl" table--attribute='aria-label="This is a table showing a loading state"'}}
```

**Comment by mcoker** [comment]
@mattnolting thanks! build passes now.

**Pull request merged**